misc/toml-parser/src/test/java/toml/parser/test/api/errors/KeyValuePairTest.java

```diff
@@ -106,22 +107,22 @@ public void testMultipleMissing() throws IOException {
     public void testArrayMissingComma() throws IOException {
 
         InputStream inputStream = Thread.currentThread().getContextClassLoader()
-                .getResourceAsStream("validations/syntax/array-missing-comma.toml");
+                .getResourceAsStream("validations/syntax/key-value/array/array-missing-comma.toml");
         Toml read = Toml.read(inputStream);
         List<TomlDiagnostic> diagnostics = read.getDiagnostics();
 
-        LineRange expectedLineRange = ErrorTestUtils.toLineRange(17, 17, 10, 11);
+        LineRange expectedLineRange = ErrorTestUtils.toLineRange(17, 17, 10, 10);
         TomlDiagnostic actualDiag = diagnostics.get(0);
 
         ErrorTestUtils
-                .validateDiagnostic(actualDiag, expectedLineRange, "invalid token '2'", DiagnosticSeverity.ERROR);
+                .validateDiagnostic(actualDiag, expectedLineRange, "missing comma token", DiagnosticSeverity.ERROR);
     }
 
-    @Test
+    @Test(enabled = false) //due to getExpectedTokenKind issue
```

Comment:
Let's create an issue to track these disabled tests

Reply:
#26865 Done.